generator/config/accumulator/stdDevSamp.yaml

```diff
@@ -16,28 +16,70 @@ arguments:
             - resolvesToNumber
 tests:
     -
-        name: 'Use in $group Stage'
-        link: 'https://www.mongodb.com/docs/manual/reference/operator/aggregation/stdDevSamp/#use-in--group-stage'
+        name: Use in $group Stage
+        link: https://www.mongodb.com/docs/manual/reference/operator/aggregation/stdDevSamp/#use-in--group-stage
         pipeline:
             -
                 $sample:
                     size: 100
             -
                 $group:
-                    _id: ~
+                    _id: null
```

Comment:
Noted that both `~` and `null` can be used according to the [spec](https://yaml.org/spec/1.2-old/spec.html#id2805071).